ROADMAP.md

```diff
@@ -174,7 +174,7 @@ It's already the default behavior of the `Icon` component. You will still be abl
        classes={{
   +      root: classes.root,
          checked: classes.checked,
-       }} 
```

Comment:
actually this space was necessary to fix formatting :/

Reply:
Oh, my bad!

Reply:
I have taken care of it in eba6b10b7c438d6499ff92cb78c1a957d637de47.